### PR TITLE
Updating alphaville-series-rw to the latest style

### PR DESCRIPTION
**Pull request by peteclark-ft**
i.e. alternativeIdentifiers etc.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,13 +1,13 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Financial Times
+Copyright (c) 2015 Financial Times
```
Comment (kghatton) [inline LICENSE]: really?

File: LICENSE [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 
 Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
 copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+furnished to do so, section to the following conditions:
```
Comment (kghatton) [inline LICENSE]: search-and-replace overdrive

File: README.md [hunk 3]
```diff
@@ -1,17 +1,73 @@
...
+With Docker:
+
+`docker build -t coco/alphaville-series-rw-neo4j .`
+
+`docker run -ti --env NEO_URL=<base url> coco/alphaville-series-rw-neo4j`
+
+
+All arguments are optional, they default to a local Neo4j install on the default port (7474), application running on port 8080, batchSize of 1024, graphiteTCPAddress of "" (meaning metrics won't be written to Graphite), graphitePrefix of "" and logMetrics false.
+
+NB: the default batchSize is much higher than the throughput the instance data ingester currently can cope with.
```
Comment (kghatton) [inline README.md]: Should we consider a more reasonable default value?

File: LICENSE (hunk 2 above)
Comment (peteclark-ft) [inline LICENSE]: haha - that one technically wasn't me, although I did copy the Licence from sections-rw!

File: alphavilleseries/alphavilleseries_service_test.go [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+	assert := assert.New(t)
+	seriesDriver := getAlphavilleSeriesCypherDriver(t)
+
+	alternativeIdentifiers := alternativeIdentifiers{TME: []string{tmeID}, UUIDS: []string{seriesUUID}}
+	seriesToDelete := AlphavilleSeries{UUID: seriesUUID, PrefLabel: prefLabel, AlternativeIdentifiers: alternativeIdentifiers}
+
+	assert.NoError(seriesDriver.Write(seriesToDelete), "Failed to write series")
+
+	found, err := seriesDriver.Delete(seriesUUID)
+	assert.True(found, "Didn't manage to delete series for uuid %", seriesUUID)
```
Comment (kghatton) [inline alphavilleseries/alphavilleseries_service_test.go]: Looks like message-format gremlins have returned here.

File: alphavilleseries/alphavilleseries_service_test.go [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+	}
+
+	db, err := neoism.Connect(url)
+	assert.NoError(err, "Failed to connect to Neo4j")
+	return NewCypherAlphavilleSeriesService(neoutils.StringerDb{db}, db)
+}
+
+func cleanUp(assert *assert.Assertions, uuid string, seriesDriver service) {
+	found, err := seriesDriver.Delete(uuid)
+	assert.True(found, "Didn't manage to delete series for uuid %", uuid)
```
Comment (kghatton) [inline alphavilleseries/alphavilleseries_service_test.go]: And here.

**Comment by kghatton** [comment]
:+1:

**Pull request merged**